screenpipe-audio/src/stt.rs

```diff
@@ -155,6 +156,7 @@ pub async fn create_whisper_channel(
     output_path: &PathBuf,
     vad_sensitivity: VadSensitivity,
     languages: Vec<Language>,
+    audio_devices_control: Option<Arc<DashMap<AudioDevice, DeviceControl>>>,
```

Comment:
why is audio device control has to change speech to text code instead of audio device recording

Reply:
also does it work well with the audio streaming api

if i stop a device